### PR TITLE
feat(slider): add `range` mode accessibility support

### PR DESCRIPTION
**Pull request by wsuwt**
## Description
- Add Arrow keys navigation support in `range` mode
- Add `Home`  and `End` support

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/slider/index.ts [hunk 1]
```diff
@@ -598,18 +598,27 @@ export class Slider extends ControlElement {
       return;
     }
 
+    this.changedThumb = event.target as HTMLDivElement;
+    const thumbName = this.changedThumb.getAttribute('name') as SliderDataName;
```
Comment (goremikins) [inline packages/elements/src/slider/index.ts]: Can you detect the thumb name in different name, e.g. if you hold the reference to thumb:
```
private getThumbName (target: EventTarget): SliderDataName | null {
  if (target === this.thumbFromRef.value) {
     return SliderDataName.from;
  }
  // return ....
}
```

File: packages/elements/src/slider/index.ts [hunk 2]
```diff
@@ -833,18 +884,43 @@ export class Slider extends ControlElement {
...
+    }
+  }
+
+  /**
+   * Validate from or to value and re-asssign
+   * @param value from or to value
+   * @param type from or to
+   * @returns validated from or to
+   */
+  private validateRange (value: string, type: SliderDataName): string {
```
Comment (goremikins) [inline packages/elements/src/slider/index.ts]: Maybe this function needs better comment. I am not sure what does it try to validate

Comment (wsuwt) [reply packages/elements/src/slider/index.ts]: It was a wrapper function of `validateFrom` and `validateTo`. I've decided to remove it and improve code comment.

File: packages/elements/src/slider/index.ts (hunk 1 above)
Comment (wsuwt) [reply packages/elements/src/slider/index.ts]: Added ref to thumbs and refactor a bit on thumb related logics.

**Review by Sarin-Udompanish — APPROVED** [review]

**Pull request merged**